Source/WebCore/dom/Document.h

```diff
@@ -1027,7 +1027,6 @@ class Document
         DOMNodeRemovedFromDocument = 1 << 3,
         DOMNodeInsertedIntoDocument = 1 << 4,
         DOMCharacterDataModified = 1 << 5,
-        OverflowChanged = 1 << 6,
```

Comment:
Should I change others to << 6 onward now?

Reply:
Yes.